portality/events/consumer_utils.py

```diff
@@ -0,0 +1,23 @@
+from portality import models
+from portality.bll import exceptions
+from portality.core import app
+
+
+def parse_application(application_raw: dict) -> models.Application:
+    try:
+        return models.Application(**application_raw)
+    except Exception as e:
+        raise exceptions.NoSuchObjectException(
+            "Unable to construct Application from supplied source - data structure validation error, {x}".format(x=e)
+        )
+
+
+def parse_email_issns(issns: list) -> str:
+    issns = issns or []
+    return ", ".join(issn for issn in issns)
+
+
+def is_enable_publisher_email() -> bool:
+    # TODO: in the long run this needs to move out to the user's email preferences but for now it
+    # is here to replicate the behaviour in the code it replaces
+    return app.config.get("ENABLE_PUBLISHER_EMAIL", False)
```

Comment:
I would place this on the user account model, and call it `is_email_enabled`, and do a check for both publisher role and this configuration option.  That will form a useful basis to build out this function later as the comment suggests.

It could also be a class method and take the account_id if needed, depending on the context

Reply:
That's in `portality.models.account`

Reply:
move to `portality.models.account` for now, but not recommend to put business logic in data layout, because complex data layout can have problems like high dependency, circle import, difficult to be extract and reuse. 

`AccountService` should be a good place for some where now general that `consumer_utils`

Reply:
I disagree with this approach @richard-jones , see comment https://github.com/DOAJ/doaj/pull/2357/files/8e54c98cb20e82e9befefeba4b2cc96aae20f7eb#diff-da6267f143e247633ae29f8348a2fe67ffef3f56963b1d36bdbb863e5fab914a